src/app/pages/StoryPage/index.jsx

```diff
@@ -195,6 +196,9 @@ const StoryPage = ({ pageData }) => {
           as="main"
           role="main"
         >
+          {pathname === '/testInclude' ? (
+            <IncludeTestForm pathWithGetParameters={location.search} />
+          ) : null}
```

Comment:
Not very happy that we have to modify the _real_ story page to add this in but couldn't think of another way to add the form to the page to get the 'rapid test' UI we were after. I guess the `pathname === '/testInclude'` can't really pass in any production scenario?

We also found `location.search` contained the path with get params: `/testInclude?testUrl=idt2`. Not sure if this is the best way of getting the get params?

Reply:
@andrewscfc Could we potentially create a story for this in story book instead and we can use a knob to input the URL instead of inserting a form onto the existing STY page? Haven't fully looked through the PR to see if that's possible so I'm not sure, and I'm also assuming this is for manual testing purposes?

Reply:
@HarveyPeachey great suggestion, we could potentially but I think the script relocation bit will have to be handled first

Reply:
@HarveyPeachey , I considered the form allowing it be changed clientside without a refresh (but not a storybook story, that's a better idea!) This would indeed need the script extraction and relocation to completed first.

The problem I see with that is we want a full stack test imo, so it needs to go through the pre-processor rules and reach the page via a SSR and work correctly. Any other test is not as 'full stack' and may miss something such as the ecoding issue @jamesdonoh has been working on.